cl/api/templates/webhooks-docs-vlatest.html

```diff
@@ -419,6 +419,14 @@ <h2 id="change-log">Change Log</h2>
       <p><strong>v1</strong> First release
       </p>
     </li>
+    <li>
+      <p><strong>v2</strong> - This release introduces support for Case Law Search Alerts results with nested documents.</p>
+      <p>You can now select the webhook version when configuring an endpoint. For most event types, there are no differences between <code>v1</code> and <code>v2</code>, as the payload remains unchanged.
+      </p>
+      <p>However, for the Search Alert event type, the <code>v2</code> payload now includes results with nested documents for search types that support it, such as Case Law. If you select <code>v1</code>, the payload remains unchanged from the previous version.
+      </p>
+      <p>For other search types, such as Oral Arguments, the payload in <code>v2</code> is identical to that in <code>v1</code>.</p>
```

Comment:
I think this could be more clear by saying:

> In these types of search responses, v1 and v2 is identical. In these other types, v2 has nested results. 

We should also say that this is building on v4 of the API and that the nested results are based on the changes arriving there.

Reply:
Sure. I've updated the texts:

--------
**v2** - This release introduces support for Case Law Search Alerts results with nested documents.

You can now select the webhook version when configuring an endpoint. For most webhook event types, there are no differences between v1 and v2, as the payload remains unchanged.

In the Search Alert webhook event type, the Oral Arguments search response remains identical between v1 and v2.

For Case Law and RECAP v2 now includes nested results, which are based on the new changes introduced in v4 of the [Search API.](/help/api/rest/v4/search/)

--------

I added `and RECAP` behind the recap alerts flag so it is shown only once RECAP alerts are released to the public.

Reply:
Also in `/help/api/rest/search/` I updated the `o` type to describe that it displays also nested documents:

![Screenshot 2025-01-03 at 1 30 03 p m](https://github.com/user-attachments/assets/75717fa7-bfd6-4c10-bcb3-9b4ba0c6200f)

Previously it only was "Case law opinions"